### ISSUE TITLE
Plotting orientation color scheme oM

### ISSUE THREAD

**Issue by DavidMainprice** [mtex-toolbox/mtex#92]
```matlab
%% this defines a color mapping for the Forsterite phase
oM = ipdfHSVOrientationMapping(ebsd('Forsterite'))
% this is the colored fundamental sector
plot(oM)
```
This no longer works in MTEX 4.1.1, may be earlier version too ? Has Syntax changed ?

all the best David

**Comment by ralfHielscher** [comment]
Hi David,

it should work. What error message do you get?

Ralf.

**Comment by DavidMainprice** [comment]
Dear Ralf,

No error, just plot of empty black outline of IPF !

all the best David

**Comment by DavidMainprice** [comment]
Dear Ralf,

The fix seems to be below, but we did not need this fix before.

all the best David

%% Orientation IPF colourbar plot - Orthorhombic with Forsterite symmetry and r = x = 1,0,0 
close all
figure
plot(oM)
set(gcf,'renderer','zbuffer')

**Comment by bjoerneske** [comment]
Same problem for me. Maybe it relates to Matlab version?

**Comment by DavidMainprice** [comment]
I do not think so, the correction suggested by Zachary Michels worked 

that is

[grains,ebsd2.grainId,ebsd2.mis2mean] = calcGrains(ebsd2('indexed'),'angle',15*degree)

I also tried

[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd2('indexed'),'angle',15*degree)

which also worked and allows to write over the original ebsd.

all the best David

**Comment by ralfHielscher** [comment]
This should been fixed since MTEX 4.1.3.

**Comment by bjoerneske** [comment]
Yes it works now, after I updated Matlab, Mtex and Microsoft SDK.
8. okt. 2015 10.03 a.m. skrev "Ralf Hielscher" <notifications@github.com>:
